### ISSUE TITLE
Context Menus - add keyboard navigation within a menu

### ISSUE THREAD

**Issue by eug3nix** [commandprompt/pgmanage#746]
Currently there are two issues with context menus:
1. When context menu is shown with a mouse right-click it it does not have a keyboard focus.
2. The user should be able to navigate the active context menu using keyboard errors and the enter key to select/activate menu item

**Comment by eug3nix** [comment]
verified
